### PR TITLE
[Docs] Update readme

### PR DESCRIPTION
**Pull request by fangyixiao18**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -70,42 +70,22 @@ MMClassification 是一款基于 PyTorch 的开源图像分类工具箱，是 [O
...
-
-同时，新版本中存在一些与旧版本不兼容的修改。请查看[迁移文档](https://mmclassification.readthedocs.io/zh_CN/1.x/migration.html)来详细了解这些变动。
-
-新版本的公测将持续到 2022 年末，在此期间，我们将基于 `1.x` 分支进行更新，不会合入到 `master` 分支。另外，至少
-到 2023 年末，我们会保持对 0.x 版本的维护。
-
-发布历史和更新细节请参考 [更新日志](https://mmclassification.readthedocs.io/zh_CN/1.x/notes/changelog.html)
+同时，新版本中存在一些与旧版本不兼容的修改。请查看 [迁移文档](https://mmclassification.readthedocs.io/zh_CN/1.x/migration.html) 来详细了解这些变动。
+ 
+发布历史和更新细节请参考 [更新日志](https://mmclassification.readthedocs.io/zh_CN/1.x/notes/changelog.html)。
```
Comment (Ezra-Yu) [inline README_zh-CN.md:88]: 1.x  ->  pretrain ?

File: README_zh-CN.md [hunk 2]
```diff
@@ -126,79 +106,123 @@ mim install -e .
...
 - [学习配置文件](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/config.html)
-- [如何微调模型](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/finetune.html)
-- [分析工具](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/analysis.html)
-- [可视化工具](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/visualization.html)
-- [其他工具](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/useful_tools.html)
+- [准备数据集](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/dataset_prepare.html)
+- [使用现有模型推理](https://mmclassification.readthedocs.io/zh_CN/1.x/user_guides/inference.html)
+- [训练](https://mmclassification.readthedocs.io/zh_CN/pretrain/user_guides/train.html)
+- [测试](https://mmclassification.readthedocs.io/zh_CN/pretrain/user_guides/test.html)
+- [下游任务](https://mmclassification.readthedocs.io/zh_CN/pretrain/user_guides/downstream.html)
```
Comment (Ezra-Yu) [inline README_zh-CN.md:114]: why some of the links use 1.x, some of links use pretrain

File: README_zh-CN.md [hunk 3]
```diff
@@ -126,79 +106,123 @@ mim install -e .
...
+  <tbody>
+    <tr align="center" valign="bottom">
+      <td>
+        <b>支持的主干网络</b>
+      </td>
+      <td>
+        <b>自监督学习</b>
+      </td>
+      <td>
+        <b>图像检索</b>
```
Comment (Ezra-Yu) [inline README_zh-CN.md:135]: I task here image retrieval should be removed, as pre-train 3rd package, image retrieval is a downstream function currently pre-trained or classified. We could add a column for others and put some training tips—for example ArcFace, RandAug, AutoAug, RepeatAugSamper, etc.

File: README_zh-CN.md [hunk 4]
```diff
@@ -219,7 +243,7 @@ MMClassification 是一款由不同学校和公司共同贡献的开源项目。
 - [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab 计算机视觉基础库
 - [MIM](https://github.com/open-mmlab/mim): MIM 是 OpenMMlab 项目、算法、模型的统一入口
 - [MMEval](https://github.com/open-mmlab/mmeval): 统一开放的跨框架算法评测库
-- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab 图像分类工具箱
+- [MMPreTrain](https://github.com/open-mmlab/mmpretrain): OpenMMLab 视觉预训练工具箱
```
Comment (Ezra-Yu) [inline README_zh-CN.md:246]: Should we delete mmselfsup to avoid it taking away some of the traffic?

File: README_zh-CN.md (hunk 2 above)
Comment (fangyixiao18) [reply README_zh-CN.md:114]: I just add the link and plan to update all readthedocs links after the readthedoc being renamed to mmpretrain

File: README_zh-CN.md (hunk 1 above)
Comment (fangyixiao18) [reply README_zh-CN.md:88]: will be updated after readthedocs being ready

File: README_zh-CN.md (hunk 4 above)
Comment (fangyixiao18) [reply README_zh-CN.md:246]: maybe keep it for sometime, we have added mmpretrain's link in mmselfsup

File: README.md [hunk 5]
```diff
@@ -59,55 +63,41 @@
 
 ## Introduction
 
-English | [简体中文](/README_zh-CN.md)
-
-MMClassification is an open source image classification toolbox based on PyTorch. It is
-a part of the [OpenMMLab](https://openmmlab.com/) project.
+MMPreTrain is an open source visual pre-training toolbox based on PyTorch. It is a part of the [OpenMMLab](https://openmmlab.com/) project.
```
Comment (tonysy) [inline README.md:66]:
```suggestion
MMPreTrain is an open source pre-training toolbox based on PyTorch. It is a part of the [OpenMMLab](https://openmmlab.com/) project.
```

**Comment by fangyixiao18** [comment]
img_v2_b122a3f9-be96-433d-a768-3d845a49b84g

File: README.md [hunk 6]
```diff
@@ -59,55 +63,41 @@
...
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/9102141/87268895-3e0d0780-c4fe-11ea-849e-6140b7e0d4de.gif" width="70%"/>
-</div>
+The `main` branch works with **PyTorch 1.8+**.
 
 ### Major features
 
 - Various backbones and pretrained models
-- Bag of training tricks
+- Bag of training strategies and tricks
```
Comment (tonysy) [inline README.md:73]:
```suggestion
- Rich training strategies(supervised learning, self-supervised learning etc).
- Bag of training tricks
```

File: README.md [hunk 7]
```diff
@@ -59,55 +63,41 @@
...
 - Large-scale training configs
 - High efficiency and extensibility
 - Powerful toolkits
```
Comment (tonysy) [inline README.md:76]:
```suggestion
- Powerful toolkits for model analysis and experiments
```

File: README.md [hunk 8]
```diff
@@ -59,55 +63,41 @@
...
+- Reproduce the training accuracy of **ConvNeXt** and **RepVGG**.
+- Support **multi-task** training and testing.
+- Support Test-time Augmentation.
+- Upgrade API to get pre-defined models of MMClassification.
+- Refactor BEiT backbone and support v1/v2 inference.
 
 This release introduced a brand new and flexible training & test engine, but it's still in progress. Welcome
 to try according to [the documentation](https://mmclassification.readthedocs.io/en/1.x/).
 
 And there are some BC-breaking changes. Please check [the migration tutorial](https://mmclassification.readthedocs.io/en/1.x/migration.html).
```
Comment (tonysy) [inline README.md:98]: The url should be updated

File: README.md [hunk 9]
```diff
@@ -222,7 +265,7 @@ This project is released under the [Apache 2.0 license](LICENSE).
 - [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab foundational library for computer vision.
 - [MIM](https://github.com/open-mmlab/mim): MIM installs OpenMMLab packages.
 - [MMEval](https://github.com/open-mmlab/mmeval): A unified evaluation library for multiple machine learning libraries.
-- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab image classification toolbox and benchmark.
+- [MMPreTrain](https://github.com/open-mmlab/mmpretrain): OpenMMLab visual pre-training toolbox and benchmark.
```
Comment (tonysy) [inline README.md:268]:
```suggestion
- [MMPreTrain](https://github.com/open-mmlab/mmpretrain): OpenMMLab pre-training toolbox and benchmark.
```

File: README_zh-CN.md [hunk 10]
```diff
@@ -57,55 +61,40 @@
...
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/9102141/87268895-3e0d0780-c4fe-11ea-849e-6140b7e0d4de.gif" width="70%"/>
-</div>
+`主分支`代码目前支持 PyTorch 1.8 以上的版本。
 
 ### 主要特性
 
 - 支持多样的主干网络与预训练模型
-- 支持配置多种训练技巧
+- 支持配置多种训练策略和技巧
```
Comment (tonysy) [inline README_zh-CN.md:71]:
```suggestion
- 支持多种训练策略（有监督学习，无监督学习等）
- 提供多种训练技巧
```

File: README_zh-CN.md [hunk 11]
```diff
@@ -219,7 +261,7 @@ MMClassification 是一款由不同学校和公司共同贡献的开源项目。
 - [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab 计算机视觉基础库
 - [MIM](https://github.com/open-mmlab/mim): MIM 是 OpenMMlab 项目、算法、模型的统一入口
 - [MMEval](https://github.com/open-mmlab/mmeval): 统一开放的跨框架算法评测库
-- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab 图像分类工具箱
+- [MMPreTrain](https://github.com/open-mmlab/mmpretrain): OpenMMLab 视觉预训练工具箱
```
Comment (tonysy) [inline README_zh-CN.md:264]:
```suggestion
- [MMPreTrain](https://github.com/open-mmlab/mmpretrain): OpenMMLab 深度学习预训练工具箱
```

**Review by tonysy — CHANGES_REQUESTED** [review]

**Review by Ezra-Yu — APPROVED** [review]
LGTM.

**Pull request merged**